kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -35,6 +36,20 @@ function collect_deployment_and_operations
 # Collect deployment logs always, even if the script ends with an error
 trap collect_deployment_and_operations EXIT
 
+###
+#   <summary>
+#       Cleanups up the GPU Drivers. This is needed when DVM is deployed with AKS Image that contains these drivers
+#   </summary>
+#   <returns>None</returns>
```

Comment:
You need to make changes in azuredeploy.json as well for this change to apply.

Reply:
https://github.com/msazurestackworkloads/azurestack-gallery/blob/master/kubernetes/template/DeploymentTemplates/azuredeploy.json#L430

Reply:
Yes I need to understand how to do that :)

Reply:
I am guessing we have to convert the script to base64 and update.

Reply:
yes.